src/app/account/create-account/create-account.component.ts

```diff
@@ -0,0 +1,30 @@
+import { AccountService } from './../shared/account.service';
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-create-account',
+  templateUrl: './create-account.component.html',
+  styleUrls: ['./create-account.component.css']
+})
+export class CreateAccountComponent implements OnInit {
+
+  account = {
+    name: '',
+    email: '',
+    password: ''
+  };
+
+  constructor(private accountService: AccountService) { }
+
+  ngOnInit() {
+  }
```

Comment:
Dei uma lida na documentação pra entender o que é esse cara, acho que poderia apagar aqui se for ficar vazio mesmo, até o construtor inclusive.

Reply:
não tem problema deixar, o componente criou como padrão eles, não tem problema nao implementar nada porenquanto